test/commands/data/bulk/results.nut.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { writeFile } from 'node:fs/promises';
+import path from 'node:path';
+import { EOL } from 'node:os';
+import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+import { expect } from 'chai';
+import { validateCsv } from 'test/testUtil.js';
```

Comment:
when I ran NUTs, it didn't like this line, updating it to : `import { validateCsv } from '../../../testUtil.js';` worked?

Reply:
fixed, ty